components/Trade/Advanced/RightPanel/RecentTrades/index.tsx

```diff
@@ -94,7 +94,7 @@ const RecentTradesContainer = styled.div<{ displayTrades: boolean }>`
     flex-direction: column;
     border-top: 1px solid var(--color-accent);
     display: flex;
-    @media (max-height: 850px) {
+    @media (max-width: 1600px) {
```

Comment:
```suggestion
    @media (max-height: 1080px) {
```

Reply:
Otherwise it causes it to show always 
![image](https://user-images.githubusercontent.com/29347276/126920295-e885d8d3-cbfb-49a8-b771-12d0cb3d5c2e.png)